### PR TITLE
React react1 week2

### PR DESCRIPTION
**Pull request by 27ManishaPatel**

### CODE REVIEW

File: react/react1/week2/class-exercise/src/App.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React, {useState} from 'react';
+import './Counter.css';
+
+
+function App() {
+  const [count, setCount] = useState(0);
+  return (
+      <div className='counter'>
+        <h1>COUNTER</h1>
+        <button onClick={() => { setCount(count + 1) }}>Click </button>
```
Comment (annamourasobral) [inline react/react1/week2/class-exercise/src/App.js]: setCount isn't an object, it's a function, so it shouldn't be wrapped in curly braces  =)
```suggestion
        <button onClick={() =>  setCount(count + 1) }>Click </button>
```

File: react/react1/week2/to-do-app/src/Main.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import React, {useState} from "react";
+
+
+const ToDoLists = [
```
Comment (annamourasobral) [inline react/react1/week2/to-do-app/src/Main.jsx:4]: now that the app is growing is probably a good time to store this data in a file like data.js, fx. =)

File: react/react1/week2/to-do-app/src/Main.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        deadline: "Fri Sep 15 2017"
+    }
+];
+
+
+
+export default function Main() {
+    const [listItem, setListItem] = useState(ToDoLists);
+
+    const DeleteList = (id) => {
```
Comment (annamourasobral) [inline react/react1/week2/to-do-app/src/Main.jsx:27]: The naming is confusing. You're deleting an item, not a List.

File: react/react1/week2/to-do-app/src/Main.jsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        const NewList = listItem.filter(li => li.id !== id)
+        setListItem(NewList)
+    }
+
+    const CheckedList = (id)=> {
```
Comment (annamourasobral) [inline react/react1/week2/to-do-app/src/Main.jsx:32]: The same naming problem here.

File: react/react1/week2/to-do-app/src/Main.jsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+                    return{
+                        ...li, completed : !li.completed
+                    }
+                }
+                return li
+            })
+        )
+    }
+
+    function ToDoListItems (props) {
```
Comment (annamourasobral) [inline react/react1/week2/to-do-app/src/Main.jsx:45]: This should be another component.

File: react/react1/week2/to-do-app/src/Main.jsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+                <button onClick={()=> CheckedList(list.id)} className="checked-button"><div className="checked-mark"></div></button>
+                {list.description} {list.deadline} 
+              <button onClick={()=> DeleteList(list.id)} className="delete-button">Delete</button>
+              </li>
+         )
+         })}
+             </ul>
+         }
+
+    const AddList = ()=>{
```
Comment (annamourasobral) [inline react/react1/week2/to-do-app/src/Main.jsx:61]: functions should be grouped

File: react/react1/week2/to-do-app/src/App.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import Header from './Header';
```
Comment (annamourasobral) [inline react/react1/week2/to-do-app/src/App.js:2]: import React as well =)

File: react/react1/week2/to-do-app/src/Header.jsx [hunk 8]
```diff
@@ -0,0 +1,7 @@
+import Timer from './Timer'
```
Comment (annamourasobral) [inline react/react1/week2/to-do-app/src/Header.jsx:2]: import React

File: react/react1/week2/to-do-app/src/Footer.jsx [hunk 9]
```diff
@@ -0,0 +1,3 @@
+export default function Footer () {
```
Comment (annamourasobral) [inline react/react1/week2/to-do-app/src/Footer.jsx:2]: import React

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Hey, Manisha, I see improvement in your code, keep going!

I noticed that you are struggling with how to build the hierarchy of components, and I thought should be interesting to share this [article from React's documentation](https://reactjs.org/docs/thinking-in-react.html) that is very useful!

Nice styling by the way, what about centralizing the container on the screen and giving some margin to the Add button? 

That's all for today..
Happy coding! 👩🏻‍💻

File: react/react1/week2/to-do-app/src/Main.jsx (hunk 3 above)
Comment (27ManishaPatel) [reply react/react1/week2/to-do-app/src/Main.jsx:27]: yes true 🙈

File: react/react1/week2/to-do-app/src/Main.jsx (hunk 5 above)
Comment (andrii1) [inline react/react1/week2/to-do-app/src/Main.jsx:45]: you can destructure here ({ lists })

File: react/react1/week2/to-do-app/src/Header.jsx [hunk 10]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Timer from './Timer'
+export default function Header () {
+    return <header>
+        <h1>To-Do List </h1>
+        <Timer/>
```
Comment (andrii1) [inline react/react1/week2/to-do-app/src/Header.jsx:6]: some files are jsx others js :)

**Pull request merged**